README.md

```diff
@@ -31,6 +31,20 @@ There's an external project [docker-lobsters](https://github.com/utensils/docker
   On MacOS you can [install with brew](https://mariadb.com/kb/en/installing-mariadb-on-macos-using-homebrew/).
   On Windows there's an [installer](https://mariadb.org/download/?t=mariadb&p=mariadb&r=11.5.2&os=Linux&cpu=x86_64&pkg=tar_gz&i=systemd&mirror=starburst_stlouis).
 
+* Start mysql and set the `root` user password
+
+```sh
+# Start the server
+mysql.server start
```

Comment:
This should be mariadb, not mysql. I'm pretty sure the mariadb fork still aliases the mysql commands, but as we've already had folks confused about which to install I'd like to be consistent.

Reply:
👍🏻 Consistent make sense.

I'm struggling to figure out how to start mariadb without explicitly using a service manager or unfriendly command. I can run `mariadbd` but that starts a non-deamon process that does not respond to ctrl-c or backgrounding. I'm using macOS/homebrew which has recommendations. But while I imagine that's a relatively common environment, it's not the only one. Listing all the platform specific options does not sound ideal.

Do you have a better suggestion? Maybe the best thing is to link to the [Starting and Stopping MariaDB Server](https://mariadb.com/kb/en/starting-and-stopping-mariadb-automatically/) page?

Reply:
Did `mariadbd --console` not do what you wanted with giving you a server you could stop with ctrl-c?

Reply:
It did not. I was not able to ctrl-c or background. Otherwise it's pretty much what I wanted, though.

When I `kill`ed it, the server _did_ gracefully shutdown, which I thought was odd. Maybe it's something with my personal setup? Either way, I think starting it up with some official background process (and then shutting it down manually if you want) is probably the best option. I don't run mysql for anything else so that's why I was trying to do this, first.

Reply:
Ah, sorry it didn't quite work out.